packages/patternfly-3/patternfly-react-extensions/less/variables.less

```diff
@@ -0,0 +1,2 @@
+@vertical-tab-pf-color:                                              @color-pf-black-500;
+@vertical-tab-pf-active-color:                                       @color-pf-blue-300;
```

Comment:
Hi @jeff-phillips-18 , thanks for the super fast response, it looks great!

Just one little thing, could you also help update the active color to **pf-blue-400**?  thanks a lot!

Reply:
Updated code and storybook. Thanks!